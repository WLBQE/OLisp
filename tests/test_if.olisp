(define (int three) 3)
(define (int five) 5)
(define (bool b) true)

(if (> three five)
  (print "3 > 5")
  (print "3 <= 5"))

(print (if b "b is true" "b is false"))
