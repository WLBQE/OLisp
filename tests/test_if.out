3 <= 5
b is true
